blog/_posts/2023-09-11-10.2-change-collection.md

```diff
@@ -60,6 +69,99 @@ There's a [dedicated post](https://stormearthandlava.com/blog/2023/09/10/t31-fir
 
 # General Shaman Changes
 
+## Splintered Elements
+
+- {{ site.data.talent.splinter }} now grants you 20% Haste plus an additional 4% Haste per extra target hit by {{ site.data.talent.primordial_wave }}, rather than 10% Haste per target hit.
+
+> Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/guardians-of-the-dream-ptr-development-notes/1666552/6) (2023-09-27)
+
+<div class="card">
+  <div class="card-header" id="spe">
+    <div class="h2 mb-0">
+      <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#spe-body" aria-expanded="false" aria-controls="spe-body">
+        General notes
+      </button>
+    </div>
+  </div>
+  <div id="spe-body" class="collapse" aria-labelledby="spe">
+    <div class="card-body" markdown="1">
+
+- This change mostly buffs {{ site.data.talent.splinter }} in Single Target and nerfs it at 5 and more targets. This is overall a good change for builds that use Splintered as their single target was their main issue.
+
+Target count | Haste gained before | Haste gained now
+--- | --- | ---
+1 | 10% | 24%
+2 | 20% | 28%
+3 | 30% | 32%
+4 | 40% | 36%
+5 | 50% | 40%
+6 | 60% | 44%
+
+</div>
+  </div>
+</div>
+
+
+
+<br>
+
+## Further Beyond
+
+- {{ site.data.talent.fb }} can now only increase the duration of {{ site.data.talent.fb }} by up to 150% of the original duration of {{ site.data.talent.asc }}.
+
+> Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/guardians-of-the-dream-ptr-development-notes/1666552/6) (2023-09-27)
+
+<div class="accordion mb-4" id="further-beyond">
+  <div class="card">
+    <div class="card-header" id="fb">
+      <div class="h2 mb-0">
+        <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#fb-body" aria-expanded="false" aria-controls="fb-body">
+          General notes
+        </button>
+      </div>
+    </div>
+    <div id="fb-body" class="collapse" aria-labelledby="fb">
+      <div class="card-body" markdown="1">
+
+- From early testing, this seems to double {{ site.data.talent.asc }}'s uptime for build that pick {{ site.data.talent.fb }}, compared to builds who do not, meaning ~45% uptime on it.
+
+</div>
+    </div>
+  </div>
+  <div class="card">
+    <div class="card-header" id="fb-issues">
+      <div class="h2 mb-0">
+        <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#fb-issues-body" aria-expanded="false" aria-controls="fb-issues-body">
+          Known Issues
+        </button>
+      </div>
+    </div>
+    <div id="fb-issues-body" class="collapse" aria-labelledby="fb-issues">
+      <div class="card-body" markdown="1">
+
+- There's currently an issue with {{ site.data.talent.dre }} proccing not adding to the {{ site.data.talent.fb }} cap.
```

Comment:
There's currently an issue with {{ site.data.talent.dre }} procs not restting or otherwise adding to to the {{ site.data.talent.fb }} cap.